pkg/prometheus/statefulset.go

```diff
@@ -35,7 +35,7 @@ import (
 const (
 	governingServiceName     = "prometheus-operated"
 	DefaultPrometheusVersion = "v2.5.0"
-	DefaultThanosVersion     = "v0.1.0"
+	DefaultThanosVersion     = "v0.2.1"
```

Comment:
It looks like there was a change in gofmt and the tests don't like your formatting of this file. For some reason (rebase?), the changes don't show up in the diff here.

Your formatting is in the v1.10 gofmt style, but the tests want the v1.11 gofmt style.

https://github.com/golang/go/issues/26098